src/network/mcpe/handler/InGamePacketHandler.php

```diff
@@ -228,7 +228,9 @@ public function handlePlayerAuthInput(PlayerAuthInputPacket $packet) : bool{
 			$this->player->jump();
 		}
 
-		if(!$this->forceMoveSync){
+		/** @phpstan-var bool $forceMoveSync */
+		$forceMoveSync = $this->forceMoveSync;
+		if(!$forceMoveSync){
```

Comment:
wtf?

Reply:
> wtf?

This is because phpstan infers that the forceMoveSync property is false.
Would it be better to use baseline?
https://github.com/pmmp/PocketMine-MP/runs/5750637772?check_suite_focus=true

Reply:
Yes. Add it to `phpstan-bugs.neon`.

This is caused by https://github.com/phpstan/phpstan/issues/5727.